student.rb

```diff
@@ -3,7 +3,7 @@
 class Student < Person
   attr_accessor :classroom
 
-  def initialize(classroom, age, _name = 'unknown', _parent_permission: true)
+  def initialize(age, _name = 'unknown', _parent_permission: true)
     super(age, _name = 'unknown', _parent_permission: true)
```

Comment:
- [ ] There is an error with the `_parent_permission` initialized in this `Student` class. The super class which is the `Person` class has it like this `parent_permission` instead of `_parent_permission`. The issue was resolved after I replaced the  `_parent_permission` with `parent_permission` in the `Student` class. See the error I got before I resolved it. So please do the above to fix it.

<img width="517" alt="Loom_9TzzxQkKCi" src="https://user-images.githubusercontent.com/81468675/153452160-84416f48-2294-4f55-854f-51091ce860c9.png">

Reply:
Thank you. Implemented.